.github/dependabot.yml

```diff
@@ -1,11 +1,11 @@
 version: 2
 updates:
-- package-ecosystem: npm
-  directory: "/"
-  schedule:
-    interval: daily
-    time: '13:00'
```

Comment:
So this is where dependably gets it's info from...
Side request: Coooullld we maybe make this once a week to reduce the # of PR's made? Not in this PR though, just wanted to throw the idea out there

Reply:
Yeah, config is defined here: https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/about-dependabot-version-updates

I'd be open to it. Might be worth bringing it up to the team to hear people's thoughts.